public/js/action-list.js

```diff
@@ -106,6 +117,13 @@
                 url = $list.attr('data-icinga-multiselect-url') + '?(' + filters.toArray().join('|') + ')';
             }
 
+            // If the item exists, remove it first before reading it
+            if (bottomStatusBar.find('.selected-items').length !== 0) {
+                bottomStatusBar.find('.selected-items').remove();
```

Comment:
Instead of removing, why don't you update it? If you're already checking its existence, this is the more efficient way I think.